.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.testing.pytestArgs": [
```

Comment:
@magdalenajankowska  you should not commit this file, please remove it (rm it, then add it, then commit)